src/select.rs

```diff
@@ -114,12 +114,12 @@ impl<T: Clone> PromptInteraction<T> for Select<T> {
         }
 
         match key {
-            Key::ArrowUp | Key::ArrowLeft => {
+            Key::ArrowUp | Key::ArrowLeft | Key::Char('k') => {
```

Comment:
Lastly, you have to test it with filter mode on. Letters will move the cursor while entering into the filter field.

Reply:
With `.filter_mode()` on the Vim keybinds don't work for moevement, I think this is preferable to not being able to filter by `hjkl`.

Reply:
@jamie-owen Yes, I mean it's not preferable, so it lgtm